LanguageFeatures/Augmentation-libraries/augmented_expression_A04_t26_lib.dart

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion The exact result of an `augmented` expression depends on what is
+/// being augmented, but it generally follows the same rules as any normal
+/// identifier:
+/// ...
+/// - Augmenting functions: When augmenting a function, `augmented` refers to
+///   the augmented function. Tear offs are not allowed, so this function must
+///   immediately be invoked.
+///
+/// @description Checks that within an augmenting function `augmented()` invokes
+/// the augmented function with appropriate default values.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=macros
+
+augment library 'augmented_expression_A04_t26.dart';
+import '../../Utils/expect.dart';
+
+augment String topLevelFunction1() => augmented();
```

Comment:
I think this declaration _must_ declare the same formal parameters, but the must also omit the default values. That is, `([String value])`, which will be turned into `([String value = "default"])` by the merging process (so there's no error because of the missing default value).

https://github.com/dart-lang/language/blob/main/working/augmentation-libraries/feature-specification.md#augmenting-functions

Reply:
Ah, of course. Just a typo, thank you for noticing. Fixed.